fdbserver/Status.actor.cpp

```diff
@@ -1720,6 +1717,10 @@ ACTOR static Future<JsonBuilderObject> dataStatusFetcher(WorkerDetails ddWorker,
 				stateSectionObj["name"] = "healthy";
 			}
 
+			// Track the number of min replicas the storage servers in this region has. The sum of the replicas from
+			// both primary and remote region give the total number of data replicas this database currently has.
+			stateSectionObj["min_replicas_remaining"] = replicas;
```

Comment:
This is the only meaningful change in this PR.

Currently, the replica info is nested quite deeply inside region data info. We could move it a level higher if needed.